### DISCUSSION TITLE
reopen: Local database update is looking as not ending

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1130] [Fixed Issues]
Performing a local database update in the library view ends in a short time frame (as expected) but the update icon in the playback window is blinking endlessly. As a consequence no additional database update is possible ("Update in progress ...").
Workaround: Manual remove of /srv/http/data/mpd/updating

**Comment by rern** [comment]
Library update removes the file by itself once finished. Apparently not in your case which means it ended somewhere earlier.
Please try `/srv/http/bash/cmd-list.sh` and observe any errors.

**Reply by klassikmann** [reply]
Update 20221210: It looks like the problem is solved.

**Comment by EarsofTin** [comment]
I experienced the unending flashing library icon following the 221210 update which persisted after reboots. Running the command **/srv/http/bash/cmd-list.sh** via ssh fixed the problem and no errors were observed in the process.

**Reply by Sagnasty** [reply]
@EarsofTin,

Your timing could not have been better - running the command you posted solved my problem as well.

**Comment by klassikmann** [comment]
Let me reopen the problem. I'm using release 20221221 now, and the described effect (apparent endless database update) is present.
But the workaround proposed by @rern  `/srv/http/bash/cmd-list.sh` is still applicable.

**Reply by klassikmann** [reply]
rAudio 202212291: 
Problem still present. After a local database update you have to issue `/srv/http/bash/cmd-list.sh` to facilitate further updates.

**Comment by klassikmann** [comment]
rAudio 20230101:
After a first test the problem seems to be solved. I'll set the category on "Fixed issues" after some additional tests.
Thank you.
